### ISSUE TITLE
Odd rendering in ZachFOL2019

### ISSUE THREAD

**Issue by rzach** [Carnap/Carnap#65]
Go to https://carnap.io/shared/rzach@ucalgary.ca/Practice%20Problems%20VII.md
The inputs
```
C(a) -> A(a,b)
C(a,b) -> A(b)
```
render respectively as
```
 C(a → A(a ,b))
 C(a ,b) → A(b))
```
(extra space at front, space before rather than after comma, missing & extra closing parens). Only happens with more than one argument.

**Comment by gleachkr** [comment]
Shoot. Will fix.

**Comment by gleachkr** [comment]
I just deployed a more careful version of the "drop outer parentheses" renderer. ~It ought to fix this - could you check the cases that were giving you trouble?~ I missed the link you included. It looks like this is now fixed.

**Comment by rzach** [comment]
Thanks! It still makes an extra space at the beginning.

**Comment by gleachkr** [comment]
I think that's probably an effect of the fix for #43 - eventually it'd be good to have separate renderers for the truth tables, I think.

**Comment by gleachkr** [comment]
I added a small tweak to the proof renderer to clean up leading whitespace, so that alignment is preserved. But, if it's OK with you, I think I'll call this fixed. Maybe if the extra spaces create any other problems, we can open a separate issue to track the necessary work in the truthtable module.

**Comment by rzach** [comment]
Thanks!
On Mar 4, 2020, at 09:39, gleachkr <notifications@github.com<mailto:notifications@github.com>> wrote:

Closed #65<https://github.com/gleachkr/Carnap/issues/65>.
